src/components/ImageEditor.vue

```diff
@@ -84,12 +84,29 @@ export default {
 				},
 			}
 		},
+
 		defaultSavedImageName() {
 			return basename(this.src, extname(this.src))
 		},
 		defaultSavedImageType() {
 			return extname(this.src).slice(1) || 'jpeg'
 		},
+
+		hasHighContrastEnabled() {
+			const themes = OCA?.Theming?.enabledThemes || []
+			return themes.find(theme => theme.indexOf('highcontrast') !== -1)
```

Comment:
Isn't the issue broader than the hightcontrast theme? If I have a custom theme, it will also be ignored by the editor, right?

Would it be possible to check whether the theme is 'dark' or 'light' by getting the `prefers-color-scheme` of the theme? And if it is dark, then do nothing?

Reply:
> If I have a custom theme, it will also be ignored by the editor, right?

Currently we don't support custom themes, but in the future, yes.
The goal would be for custom themes to specify if they are either `highcontrast`, `lowcontrast`, `light` or `dark`.
Then we'll be able to use that. But for now that should be llright :)